### PR TITLE
GCP docs

### PR DESCRIPTION
**Pull request by nico-shishkin**

### CODE REVIEW

File: docs/_include/general-shipping/gcp.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
```
Comment (mirii1994) [inline docs/_include/general-shipping/gcp.md:1]: Can we specify in the file's name that this doc is for metrics?

File: docs/shipping/GCP/gcp-ai-platform.md [hunk 2]
```diff
@@ -14,83 +14,6 @@ metrics_alerts: []
...
-
-{@include: ../../_include/metric-shipping/telegraf-run.md}  
-  
-### Check Logz.io for your metrics
-
-Give your data some time to get from your system to ours, then log in to your Logz.io Metrics account, and open [the Logz.io Metrics tab](https://app.logz.io/#/dashboard/metrics/).
-
-
- 
+{@include: ../../_include/general-shipping/gcp.md}
```
Comment (mirii1994) [inline docs/shipping/GCP/gcp-ai-platform.md:19]: How can we both add the include, and also add the specific value for telemetry_list that will allow shipping metrics for this specific service?
This comment applies to all the documents in this PR.

**Review by mirii1994 — CHANGES_REQUESTED** [review]

**Comment by nico-shishkin** [comment]
Hi @mirii1994 , I've resolved the comments, I believe

**Comment by mirii1994** [comment]
@nico-shishkin I'm closing this PR because there's an issue with the Netlify preview, i'll open a new one instead.

**Pull request closed without merging**